fdbserver/QuietDatabase.actor.cpp

```diff
@@ -211,10 +211,11 @@ ACTOR Future<std::pair<int64_t,int64_t>> getTLogQueueInfo( Database cx, Referenc
 
 ACTOR Future<vector<StorageServerInterface>> getStorageServers( Database cx, bool use_system_priority = false) {
 	state Transaction tr( cx );
-	if (use_system_priority)
-		tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
-		tr.setOption(FDBTransactionOptions::LOCK_AWARE);
```

Comment:
This isn't directly related to my changes, but I stumbled upon this while testing and figured this was a bug so I changed it to what I think it was supposed to do. Please comment if it should behave differently.

Reply:
I think your change does not change the functionality. 
The indentation of `tr.setOption(FDBTransactionOptions::LOCK_AWARE);`  was wrong.

Your change definitely makes the code better in terms of readability.

Reply:
This also moved the option setting into the loop, which causes the options to be set again on retry. That seems right to me, and it makes me wonder whether we are sufficiently exercising retry code paths.